src/service/service_layer.h

```diff
@@ -71,6 +80,11 @@ class ServiceLayer {
   // @ret: ChirpObj constructed from `chirp`
   ChirpObj ParseChirpString(const std::string& chirp);
 
+  // Helper function to parse data from DataStore
```

Comment:
This comment should specify what `chirps` contains and what the return vector contains in relation to `chirps`.

Also, you shouldn't pass in a non-const reference.